### ISSUE TITLE
Enable testing with tox

### ISSUE THREAD

**Issue by rafguns** [rafguns/linkpred#2]
Something like waf may do the trick. I'm thinking of e.g.:

```
# Create a tarball
git archive --format=tar.gz --prefix=linkpred/ HEAD > linkpred.tar.gz

# Create an installer
pyinstaller linkpred.spec

# Do all tests and count coverage
nosetests --with-doctest --with-coverage --cover-package=linkpred
```

**Comment by rafguns** [comment]
Alternative: do this in setup.py. [This guide](http://www.jeffknupp.com/blog/2013/08/16/open-sourcing-a-python-project-the-right-way/) shows a way to do testing through `python setup.py test`.

**Comment by rafguns** [comment]
Testing as in https://github.com/rafguns/linkpred/issues/2#issuecomment-91150132 is now deprecated in favour of tox.

**Comment by rafguns** [comment]
Fixed in 65e76c6b3e40ac1b7fb95b99c5ea7b50c7598632.

To use this, first `pip install tox` and then simply run `tox`. If you want to restrict to a specific version (e.g. Python 3.6), run `tox -e py36`.
